src/mvs_eland/C1_verification.py

```diff
@@ -124,6 +127,60 @@ def check_feedin_tariff(dict_values):
     return
 
 
+def check_time_series_values_between_0_and_1(time_series):
+    r"""
+    Checks whether all values of `time_series` in [0, 1].
+
+    Parameters
+    ----------
+    time_series : pd.Series
+        Time series to be checked.
+
+    Returns
+    -------
+    bool
+        True if values of `time_series` within [0, 1], else False.
+
+    """
+    boolean = time_series.between(0, 1)
+    if boolean.all() == False:
+        return False
+    else:
+        return True
+
+
+def check_non_dispatchable_source_time_series(dict_values):
+    r"""
+    Raises error if time series of non-dispatchable sources are not between [0, 1].
+
+    Parameters
+    ----------
+    dict_values : dict
+        Contains all input data of the simulation.
+
+    Returns
+    -------
+    Indirectly, raises error message in case of time series of non-dispatchable sources
+    not between [0, 1].
+
+    """
+    # go through all non-dispatchable sources
+    for key, source in dict_values[ENERGY_PRODUCTION].items():
+        if (
+            TIMESERIES_NORMALIZED in source
```

Comment:
I think you want to search for `TIMESERIES `here - as per my understanding `TIMESERIES_NORMALIZED` should always be normalized ie. between 0,1.